### ISSUE TITLE
Order Notification not sending to Admin in Multistores

### ISSUE THREAD

**Issue by ankurverma5540** [magento/magento2#19520]
### Preconditions (*)

1. Crate Two Store views and Stores
2. Place an order from any stores. user will get order notification email but admin will not get any email

### Steps to reproduce (*)

1. Crate Two Store views and Stores
2.Place an order from any stores. user will get order notification email but admin will get any email

### Expected result (*)

1. We an order is placed, user and admin should receive email.

### Actual result (*)

1. Only User receive email but admin not getting any email.

**Comment by magento-engcom-team** [comment]
Hi @ankurverma5540. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@ankurverma5540 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by ankurverma5540** [comment]
yes

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-andrii. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by gwharton** [comment]
Looks like a duplicate of #16203

**Comment by ankurverma5540** [comment]
There is no solution in that thread

**Comment by gwharton** [comment]
The fix is there, Its just you haven't applied it correctly.

If you have no /lib/internal/Magento folder, then it will be /vendor/magento instead.

Don't copy the entire file, just locate the files that have changed, then make the necessary changes inside the files in accordance with the commit. Make sure you clean/build DI/Redeploy static content.

And yes, we are editing the core!

**Comment by ankurverma5540** [comment]
according to this commit
[9a7c9e5#diff-dfb3ee5a7f863458afea75f5524cd2e5](https://github.com/magento/magento2/commit/9a7c9e553b1c9814907105388bffc2e8a91353a5#diff-dfb3ee5a7f863458afea75f5524cd2e5)

Change in app/code/Magento/Newsletter/Test/Unit/Model/Queue/TransportBuilderTest.php
Magento/Framework/Mail/Template/TransportBuilder.php
and in this  file
Magento/Framework/Mail/Test/Unit/Template/TransportBuilderTest.php

??

**Comment by gwharton** [comment]
Those paths are for github installs, for composer installs the paths would be

/vendor/magento/newsletter/Test/Unit/Model/Queue/TransportBuilderTest.php
/vendor/magento/framework/Mail/Template/TransportBuilder.php
/vendor/magento/framework/Mail/Test/Unit/Template/TransportBuilderTest.php

**Comment by ankurverma5540** [comment]
If i made changes to 
/vendor/magento/newsletter/Test/Unit/Model/Queue/TransportBuilderTest.php
then, email not coming to admin but also admin login not working.
